src/runtime/flow_node_handler/activity_handler/activity_handler.ts

```diff
@@ -468,31 +668,13 @@ export abstract class FlowNodeHandlerInterruptible<TFlowNode extends Model.Base.
     return handlerForNextFlowNode.execute(currentProcessToken, processTokenFacade, processModelFacade, identity, boundaryInstanceId);
   }
 
-   /**
-    * Cancels and clears all BoundaryEvents attached to this handler.
-    */
-   private async _detachBoundaryEvents(token: ProcessToken, processModelFacade: IProcessModelFacade): Promise<void> {
-     for (const boundaryEventHandler of this._attachedBoundaryEventHandlers) {
-       await boundaryEventHandler.cancel(token, processModelFacade);
-     }
-     this._attachedBoundaryEventHandlers = [];
-   }
-
-   private _terminateProcessInstance(identity: IIdentity, token: ProcessToken): void {
-
-     const eventName: string = eventAggregatorSettings.messagePaths.processInstanceWithIdTerminated
-       .replace(eventAggregatorSettings.messageParams.processInstanceId, token.processInstanceId);
-
-     const message: ProcessTerminatedMessage = new ProcessTerminatedMessage(token.correlationId,
-                                                                            token.processModelId,
-                                                                            token.processInstanceId,
-                                                                            this.flowNode.id,
-                                                                            this.flowNodeInstanceId,
-                                                                            identity,
-                                                                            token.payload);
-     // ProcessInstance specific notification
-     this.eventAggregator.publish(eventName, message);
-     // Global notification
-     this.eventAggregator.publish(eventAggregatorSettings.messagePaths.processTerminated, message);
-   }
+  /**
+   * Cancels and clears all BoundaryEvents attached to this handler.
+   */
+  private async _detachBoundaryEvents(token: ProcessToken, processModelFacade: IProcessModelFacade): Promise<void> {
+    for (const boundaryEventHandler of this._attachedBoundaryEventHandlers) {
+      await boundaryEventHandler.cancel(token, processModelFacade);
+    }
+    this._attachedBoundaryEventHandlers = [];
```

Comment:
Adding a spacing here increases the readability a lot IMO

```suggestion

    this._attachedBoundaryEventHandlers = [];
```